extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/B3PropagatorInjectorMultipleHeaders.java

```diff
@@ -26,6 +26,11 @@
 
     String sampled = spanContext.isSampled() ? Common.TRUE_INT : Common.FALSE_INT;
 
+    if (B3Propagator.MULTI_HEADER_DEBUG.equals(context.get(B3Propagator.DEBUG_CONTEXT_KEY))) {
```

Comment:
Hm. In this case, aren't we putting `Common.TRUE_INT` into the context, not `B3Propagator.MULTI_HEADER_DEBUG`? They might happen to have the same value at the moment, but I don't think we should confuse the constant from the wire protocol with the constant that we're using internally for propagation.

Reply:
Ah good point - whether the b3 debug was enabled or not is sort of independent of how it was actually modeled in the header, and it might be converted from single to multiple, or vice versa. @jarebudev How about just using a `Boolean` context value for this?

Reply:
Yeh makes sense. I've changed it to set a boolean, hopefully this sorts it!